packages/core/src/Badge/Badge.spec.tsx

```diff
@@ -81,4 +82,10 @@ describe('Badge', () => {
     expect(json).toHaveStyleRule('background-color', theme.colors.lightBlue)
     expect(json).toHaveStyleRule('color', theme.colors.text)
   })
+
+  test('textTransform', () => {
+    render(<Badge textTransform='lowercase'>Lowercase Text</Badge>)
+    expect(screen.getByText('Lowercase Text')).toHaveStyleRule('text-transform', 'lowercase')
+    expect(screen.getByText('Lowercase Text')).toHaveStyleRule('letter-spacing', 'normal')
+  })
```

Comment:
In the future, you don't necessarily need to write a unit test as long as you add a new story that demonstrates the functionality since we'll see any diffs in Chromatic.